specs/workload/Workload.md

```diff
@@ -0,0 +1,160 @@
+# 1. Workload Management
+
+This feature provides APIs to manage a identify and manage a set of resources used collectively to
+provide and perform a workload of activity.
+
+- [1. Workload Management](#1-workload-management)
+
+# 2. Background
+
+Windows can provide a set of functionality via one or more MSIX packages and other resources. This
+API provides the means to manage these related packages and resources as a higher level group of
+functionality, called a 'workload'.
+
+Microsoft-internal task [48267137](https://task.ms/48267137)
+
+# 3. Description
+
+This API supports managing workloads focusing on the following scenarios:
+
+* Enumerate installed workloads
+* Retrieve properties of a workload
+* Remove installed packages and related resources comprising a workload
+
+## 3.1. API Structure
+
+## 3.2. Workload Definition via appxmanifest.xml
+
+A workload has the following properties:
+
+* Id -- programmatic identifier
+* DisplayName -- localizable name for user/human consumption
+* Handler -- WinRT object providing operations manipulating the workload
+
+An MSIX package can define one or more workloads. The following manifest schema is used:
+
+```xml
+<Package>
+  ...
+  <Properties>
+    <workload:Workloads>
+      <Workload Id="id" DisplayName="dn">
+        <Actions>
+          <Remove ActivatableClassId="acid">
```

Comment:
Should this be `<Handler ActivatableClassId="acid">` to accomoddate additional verbs in the future (Reset, Repair, ...)